rfcs/onboard_infrastructure_providers.md

```diff
@@ -0,0 +1,48 @@
+### Introduction
+
+It seems reasonable that being Paseo a community testnet not a single party has to make all the efforts of maintaining this platform. At the same time, moving away from a single operator should not compromise the availablity of the network.
+
+This RFC talks about how different infrastructure providers can support Paseo network.
```

Comment:
This RFC (PAS) discusses about how different infrastructure providers can join and contribute to the Paseo network.